addons/account/report/account_invoice_report.py

```diff
@@ -95,7 +95,23 @@ def _select(self):
                 line.quantity / NULLIF(COALESCE(uom_line.factor, 1) / COALESCE(uom_template.factor, 1), 0.0) * (CASE WHEN move.move_type IN ('in_invoice','out_refund','in_receipt') THEN -1 ELSE 1 END)
                                                                             AS quantity,
                 -line.balance * currency_table.rate                         AS price_subtotal,
-                line.price_total * (CASE WHEN move.move_type IN ('in_invoice','out_refund','in_receipt') THEN -1 ELSE 1 END)
+                line.price_total *  (CASE WHEN move.move_type IN ('in_invoice','out_refund','in_receipt') THEN -1 ELSE 1 END) *
+                  COALESCE(
+                    (
+                      SELECT rate
+                      FROM res_currency_rate
+                      WHERE name <= line.date AND currency_id = line.company_currency_id
+                      ORDER BY name DESC
+                      LIMIT 1
+                    ) / (
+                      NULLIF((
+                        SELECT rate
+                        FROM res_currency_rate
+                        WHERE name <= line.date AND currency_id = line.currency_id
+                        ORDER BY name DESC
+                        LIMIT 1
+                        ), 0
+                  )), 1.0)
```

Comment:
I'm wondering if it doesn't just miss the `* currency_table.rate` that is on the line `price_subtotal` above and on the line `price_average` below